johnsonandjohnson/Bodiless-JS
# feat(components): Implement Interface for Managing GTM Datalayer.

### Changes:
- Add an API to allow for composing user interface for GTM datalayers.

### Test Instructions:
- Enabled google tag manager. 
- Visit the home page on example/test-site.
- Note the GTM form appears.
- Note the GTM form had page type field.
- Visit a product page.
- Note the product page has example properties defined such as sku, name, variant.
- Edit and save the fields.
- Disable Edit mode.
- Inspect the head element for the appropriate datalayer script with element. 
 
### Related Issues
Fixes #451
